package.json

```diff
@@ -33,6 +34,7 @@
   "dependencies": {
     "@bcoe/v8-coverage": "^0.2.3",
     "@istanbuljs/schema": "^0.1.2",
+    "@types/istanbul-lib-report": "^3.0.0",
```

Comment:
How complex is the watermark type, I wonder if we could get away with not pulling in the `@types/istanbul-lib-report` dependency, and just including the description inline.

Reply:
thats fair, inlined and removed the dep